V/VCDiff/build_tarballs.jl

```diff
@@ -0,0 +1,55 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "VCDiff"
+version = v"0.8.4"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/google/open-vcdiff.git", "868f459a8d815125c2457f8c74b12493853100f9")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/open-vcdiff/
+mkdir build
+git submodule update --init --recursive
+cd $prefix
+cmake -Dvcdiff_build_exec=OFF -DBUILD_SHARED_LIBS=ON $WORKSPACE/srcdir/open-vcdiff/
+make
+make install
+exit
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = [
+    Platform("i686", "linux"; libc = "glibc"),
+    Platform("x86_64", "linux"; libc = "glibc"),
+    Platform("aarch64", "linux"; libc = "glibc"),
+    Platform("armv6l", "linux"; call_abi = "eabihf", libc = "glibc"),
+    Platform("armv7l", "linux"; call_abi = "eabihf", libc = "glibc"),
+    Platform("powerpc64le", "linux"; libc = "glibc"),
+    Platform("i686", "linux"; libc = "musl"),
+    Platform("x86_64", "linux"; libc = "musl"),
+    Platform("aarch64", "linux"; libc = "musl"),
+    Platform("armv6l", "linux"; call_abi = "eabihf", libc = "musl"),
+    Platform("armv7l", "linux"; call_abi = "eabihf", libc = "musl"),
+    Platform("x86_64", "freebsd"; )
+]
```

Comment:
Can try to build for all platforms?  Also, the log [shows](https://buildkite.com/julialang/yggdrasil/builds/3611#0188da09-6711-40fb-ab91-4268b9d2fb7b/655-952) that you need to [expand the C++ string ABI](https://docs.binarybuilder.org/stable/building/#Expanding-C-string-ABIs-or-libgfortran-versions)
```suggestion
platforms = supported_platforms()
platforms = expand_cxxstring_abis(platforms)
```